src/app.module.ts

```diff
@@ -1,16 +1,16 @@
-import { Organization } from './organizations/entities/organization.entity';
-import { UsersModule } from './users/users.module';
+import { Organization } from './models/organizations/entities/organization.entity';
+import { UsersModule } from './models/users/users.module';
 import { ApolloDriver, ApolloDriverConfig } from '@nestjs/apollo';
 import { Module } from '@nestjs/common';
 import { GraphQLModule } from '@nestjs/graphql';
 import { join } from 'path';
 import { SequelizeModule } from '@nestjs/sequelize';
-import { User } from './users/entities/user.entity';
-import { OrganizationsModule } from './organizations/organizations.module';
-import { OrdersModule } from './orders/orders.module';
-import { Order } from './orders/entities/order.entity';
-import { ConfigModule } from '@nestjs/config';
-import { AuthModule } from './auth/auth.module';
+import { User } from './models/users/entities/user.entity';
```

Comment:
Read more about aliases in typescript https://christianlydemann.com/simpler-typescript-paths-with-path-aliases/

Reply:
already done)